core-test/src/test/scala/org/bitcoins/core/crypto/TransactionSignatureCheckerTest.scala

```diff
@@ -1,8 +1,229 @@
 package org.bitcoins.core.crypto
 
+import org.bitcoins.core.currency.CurrencyUnits
+import org.bitcoins.core.number.UInt32
+import org.bitcoins.core.policy.Policy
+import org.bitcoins.core.protocol.script.ScriptPubKey
+import org.bitcoins.core.protocol.transaction._
 import org.bitcoins.testkit.util.BitcoinSUnitTest
 
 /**
   * Created by chris on 2/29/16.
   */
-class TransactionSignatureCheckerTest extends BitcoinSUnitTest {}
+class TransactionSignatureCheckerTest extends BitcoinSUnitTest {
+
+  behavior of "TransactionSignatureChecker"
+
+  it must "validate a P2PK sig" in {
```

Comment:
for all of these validation tests you should add invalidation tests or asserts for if something small is changed